src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes.csproj

```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.3.1" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.VisualStudio.Validation" Version="16.9.32" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.2.0" PrivateAssets="all" />
```

Comment:
This will need to be rolled back as well.

Reply:
Done